.redhat-ci.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -xeuo pipefail
+
+# F25 currently has 2.2.1, so install from pypi
+pip install ansible==2.2.2.0
+
+# do a simple ping to make sure the nodes are available
+ansible -vvv -i .redhat-ci.inventory nodes -a 'rpm-ostree status'
```

Comment:
Well, it's both a ping as well as getting information about the hosts to know what they're running, which canonically on AH is `rpm-ostree status`. I'll update the comment to make this more obvious.